puterbot/strategies/summary_mixin.py

```diff
@@ -0,0 +1,105 @@
+"""
+Implements a ReplayStrategy mixin which identifies whether windows are different
+from summarizing their contents.
+
+Usage:
+
+    $ python puterbot/summary_mixin.py SummaryReplayStrategyMixin
+"""
+from puterbot.models import Recording
+from puterbot.strategies.base import BaseReplayStrategy
+
+
+from sumy.summarizers.lsa import LsaSummarizer
+from sumy.parsers.plaintext import PlaintextParser
+from sumy.nlp.tokenizers import Tokenizer
+from sumy.nlp.stemmers import Stemmer
+from sumy.utils import get_stop_words
+from nltk.corpus import wordnet
+import re
+from fuzzywuzzy import fuzz
+import statistics
+
+
+class SummaryReplayStrategyMixin(BaseReplayStrategy):
+    """
+    The summary of text ReplayStrategyMixin.
+    """
+
+    def __init__(
+        self,
+        recording: Recording,
+    ):
+        """
+        See base class.
+        """
+        super().__init__(recording)
+
+    def get_summary(
+        self,
+        ascii_curr: str,
+        ascii_prev: str,
+        ocr_curr: str,
+        ocr_prev: str,
+    ) -> float:
+        """Returns how similar the contents of 2 screenshots are.
```

Comment:
Hi @dianzrong , it seems that the method name does not match the behavior of the function or the docstring. `get_summary` is something we want, but measuring how similar the contents are based on the summary of the text contained in them is not.

Can you please modify this to simply return the summary of some text that is passed in? No need for ASCII or OCR anywhere in this file.